frontend_lib/src/component/DropdownMenu/DropdownMenu.jsx

```diff
@@ -50,7 +50,7 @@ DropdownMenu.propTypes = {
   buttonDisabled: PropTypes.bool,
   buttonIcon: PropTypes.string,
   buttonImage: PropTypes.string,
-  buttonLabel: PropTypes.string,
+  buttonLabel: PropTypes.oneOfType([PropTypes.string, PropTypes.component]),
```

Comment:
You could have used buttonOpts instead of buttonLabel, but this works too.

Reply:
Yes, I'll do that.